app/javascript/tests/public_health/bulk_actions/CloseRecords.test.js

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import CloseRecords from '../../../components/public_health/bulk_actions/CloseRecords';
+import { mockPatient1, mockPatient2, mockPatient6 } from '../../mocks/mockPatients';
+import { mockMonitoringReasons } from '../../mocks/mockMonitoringReasons';
+
+const onCloseMock = jest.fn();
+const mockToken = 'testMockTokenString12345';
+
+function getWrapper(patientsArray) {
+  let wrapper = shallow(<CloseRecords authenticity_token={mockToken} patients={patientsArray} close={onCloseMock} monitoring_reasons={mockMonitoringReasons} />);
```

Comment:
you can hardcode `patientsArray` here  as `[mockPatient1, mockPatient2, mockPatient6]` since it's always the same as it's passed in.  Maybe declare it as a const at the top and then just use that as the prop